src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -41,15 +41,25 @@ public MarkdownV2Renderer(ParserMode mode, int maxSyntaxWidth)
 
         public string MamlModelToString(MamlCommand mamlCommand, bool skipYamlHeader)
         {
-            return MamlModelToString(mamlCommand, null, skipYamlHeader);
+            return MamlModelToString(mamlCommand, null, skipYamlHeader, false);
+        }
+
+        public string MamlModelToString(MamlCommand mamlCommand, bool skipYamlHeader, bool detailedInput)
+        {
+            return MamlModelToString(mamlCommand, null, skipYamlHeader, detailedInput);
         }
 
         public string MamlModelToString(MamlCommand mamlCommand, Hashtable yamlHeader)
         {
-            return MamlModelToString(mamlCommand, yamlHeader, false);
+            return MamlModelToString(mamlCommand, yamlHeader, false, false);
         }
 
-        private string MamlModelToString(MamlCommand mamlCommand, Hashtable yamlHeader, bool skipYamlHeader)
+        public string MamlModelToString(MamlCommand mamlCommand, Hashtable yamlHeader, bool detailedInput)
```

Comment:
wow that such a zoo of overloads. Can we refactor it? Maybe remove some of them with fewer parameters or create a `ModelToStringInfo` class that would allow caller to pass all parameters in the combinations they want.
The goal is not to have overloads with the same number of arguments that only differ by type. For this api, it would be too confusing.